src/components/flowchart/styles/_variables.scss

```diff
@@ -1,67 +1,50 @@
 @import '../../../styles/variables';
 
 /**
- * List of colour variables for the different parts of the flowchart
+ * Themed colour variables for the flowchart component
  */
 
-//-------------- LIGHT --------------//
-
-// Icon
-
-$node-icon-fill-light: $color-text-light;
-
-// Node bg
-
-$node-fill-light-default: #f0f1f3;
-$node-stroke-light-default: #c6cace;
-$node-fill-light-active: #c6e6fb;
-$node-stroke-light-active: #b2cada;
-$node-fill-light-selected: #aad8ff;
-$node-stroke-light-selected: $color-primary;
-
-// Label
-
-$node-labeltext-fill-light: $color-text-light;
-
-// Edge
-
-$edge-light: #888;
-$edge-arrowhead-light: #2a3b46;
-
-// Layer
-
-$layer-light: mix($color-bg-light-1, $color-bg-light-2);
-
-// Print/export styling
-
-$node-fill-light-export: #f0f1f3;
-$node-stroke-light-export: #6b6a6b;
-$edge-light-export: #686768;
-
-//-------------- DARK --------------//
-
-// Icon
-
-$node-icon-fill-dark: $color-text-dark;
-
-// Node bg
-
-$node-fill-dark-default: #2a3b46;
-$node-stroke-dark-default: #354854;
-$node-fill-dark-active: #124066;
-$node-stroke-dark-active: #234f73;
-$node-fill-dark-selected: #004d8c;
-$node-stroke-dark-selected: $color-primary;
-
-// Label
-
-$node-labeltext-fill-dark: $color-text-dark;
-
-// Edge
-
-$edge-dark: #666;
-$edge-arrowhead-dark: #cacaca;
-
-// Layer
-
-$layer-dark: mix($color-bg-dark-1, $color-bg-dark-2);
+.kui-theme--light {
+  --node-icon-fill: #{$color-text-light};
+  --node-fill-default: #f0f1f3;
```

Comment:
In following the pattern of referring to colors using variables, this color should be referred by the variable`$color-bg-light-3`?

Reply:
ah yeah interesting, I hadn't caught that some of these are duplicates of the core ones. It's a preexisting issue so slightly out of scope of this PR but I don't mind fixing it here 🌞

Reply:
fixed